eng/versioning/external_dependencies.txt

```diff
@@ -398,3 +398,6 @@ storage_com.microsoft.azure:azure-storage;8.4.0
 
 # sdk\appconfiguration\azure-spring-cloud-test-appconfiguration-config\pom.xml
 spring_com.microsoft.azure:azure;1.34.0
+
+# sdk\spring\spring-cloud-azure-service\pom.xml
+spring_redis.clients:jedis;3.8.0
```

Comment:
@Netyyyy will this version be managed by our upgrading script?

Reply:
Yes, it was. 
Refer to https://github.com/Azure/azure-sdk-for-java/blob/51f5312204f163f3f51b24ad8317e00d75617779/sdk/spring/scripts/spring_boot_2.7.7_managed_external_dependencies.txt#L1550
it was managed at least in spring-boot 2.7.7

Reply:
But this version has a prefix `spring_`. Will this be managed too?

Reply:
If prefixed, it will not be managed by spring, only here.

Reply:
@Netyyyy let's discuss offline to see how to manage this.